docs/features/checks.rst

```diff
@@ -35,9 +42,10 @@ Overriding Quality Checks
 -------------------------
 
 It is possible to override the quality check if the translation is correct.
```

Comment:
I would suggest to replace all appearances of "override" and others by **"mute"**.